doc/man1/openssl-s_server.pod.in

```diff
@@ -644,6 +704,14 @@ This option must be provided in order to use a PSK cipher.
 Use the pem encoded SSL_SESSION data stored in I<file> as the basis of a PSK.
 Note that this will only work if TLSv1.3 is negotiated.
 
+=item B<-srpvfile>
+
+The verifier file for SRP (deprecated).
```

Comment:
I thought we'd been adding _This option is deprecated._ rather than a parenthesised word.

Reply:
I was not aware of this convention - changed.